chromadb/test/property/test_filtering.py

```diff
@@ -292,3 +296,67 @@ def test_boolean_metadata(api: API) -> None:
     res = coll.get(where={"test": True})
 
     assert res["ids"] == ["1", "3"]
+
+
+def test_fluid_filter_helper_and_eq_contains(api: API) -> None:
```

Comment:
Tests in the /property folder should be using generative approaches and the hypothesis Generations. Can we add a recursive strategy that generates the where filters and where document filters generatively and verifies they work as expected? You should be able to see the existing where strategies and borrow from those heavily.

Reply:
Not too familiar with Hypothesis but I'll give it a go.